tgui/packages/tgui/interfaces/AirAlarm.tsx

```diff
@@ -381,16 +381,16 @@ const AirAlarmControlThresholds = (props, context) => {
         <Table.Row>
           <Table.Cell bold>Threshold</Table.Cell>
           <Table.Cell bold color="bad">
-            Minimum Hazard
+            Danger below
           </Table.Cell>
           <Table.Cell bold color="average">
-            Minimum Warning
+            Warning below
           </Table.Cell>
           <Table.Cell bold color="average">
-            Maximum Warning
+            Warning above
           </Table.Cell>
           <Table.Cell bold color="bad">
-            Maximum Hazard
+            Danger above
```

Comment:
can you post a screenshot of what this looks like because i don't really think it would look good? just wanna see if this makes logical sense

Reply:
![image](https://github.com/tgstation/tgstation/assets/3625094/95a1be07-4ddf-4fbd-a358-f1a8943a2416)

The layout didn't change, but it should be more clear to people what these numbers do.